tests/specs/close-db.js

```diff
@@ -5,10 +5,17 @@
         var indexedDB = window.indexedDB || window.webkitIndexedDB || window.mozIndexedDB || window.oIndexedDB || window.msIndexedDB;
 
         beforeEach(function (done) {
-            this.dbName = guid();
+            var spec = this;
+            spec.server = undefined;
+            if (!this.dbName) {
+                this.dbName = guid();
+                done();
```

Comment:
I think I ran into a memory issue in some environment or another with the way it had been...